countries.js

```diff
@@ -0,0 +1,2782 @@
+/*eslint quotes: ["error", "double"]*/
```

Comment:
Have not seen the use of postal code format or postal code regex in the code. If this the case, then we should generate the same file without it to save some bytes. When we do decide to use it we can generate it with the added fields.

Reply:
- [ ] re-generate with no postal-code prefixes
@gannan08 having seen the script that generates the countries.js now do you want me to include it? would not that want the in the npm build, but could be excluded and used in development.

Reply:
I would create a top level directory called `/bin` and put your script in there. Then document it in the README.

Reply:
- [ ] move script into project